.gitignore

```diff
@@ -1,2 +1,3 @@
 release
 .venv
```

Comment:
Remove this and also change https://github.com/osism/cfg-generics/blob/main/environments/manager/run.sh#L17 to venv.

Reply:
Done